openrndr-core/src/main/kotlin/org/openrndr/shape/Triangulator.kt

```diff
@@ -9,7 +9,13 @@ import org.openrndr.shape.tessellation.GLUtessellatorCallbackAdapter
 import org.openrndr.shape.tessellation.Tessellator
 import java.util.*
 
+/**
+ * Cumulative sum of [Int]s in the [List].
+ *
+ * Not to be confused with [sum].
+ */
 fun List<Int>.cumsum(): List<Int> {
+    listOf(0, 3).sum()
```

Comment:
left over code?

Reply:
It probably is, but it's there and I documented it!